packages/portal-contracts/src/Portal.sol

```diff
@@ -80,11 +80,11 @@ struct Escrow {
     /**
      * @dev If correct amount is paid to script hash, who keeps the escrow?
      */
-    address successOpenEscrow;
+    address successRecipient;
```

Comment:
this fixes https://github.com/dcposch/silverportal/pull/28/files , which looks like it find-replaced `Recipient` with `OpenEscrow` everywhere?